containers/src/Kokkos_ScatterView.hpp

```diff
@@ -229,7 +229,7 @@ struct ScatterValue<ValueType, Kokkos::Experimental::ScatterSum, DeviceType,
  public:
   KOKKOS_FORCEINLINE_FUNCTION ScatterValue(ValueType& value_in)
       : value(value_in) {}
-  KOKKOS_FORCEINLINE_FUNCTION ScatterValue(ScatterValue&& other)
+  KOKKOS_FORCEINLINE_FUNCTION ScatterValue(ScatterValue&& other) noexcept
```

Comment:
Depending on the compiler this now may insert a catch on every access to a scatterview which is performance critical code ...